### ISSUE TITLE
Trigger Piece RequestL Postgres

### ISSUE THREAD

**Issue by Immortalin** [activepieces/activepieces#1454]
**Describe your usecase.**
Postgres CRUD events trigger support. End user should be able to choose which table/columns to watch and which the data should be returned. 

Supabase has a similar setup: https://supabase.com/docs/guides/realtime/extensions/postgres-changes

**Describe alternatives you've considered**
N/A

**Comment by abuaboud** [comment]
If it's about Supabase Postgres, then It's similar to this ticket right?

https://github.com/activepieces/activepieces/issues/1207

**Comment by Immortalin** [comment]
No, I am referring to Postgres in general. Supabase's realtime architecture could be used to implement the trigger for activepiece (they listen to the postgres event source via its`NOTIFY` function)
